config/routes.rb

```diff
@@ -0,0 +1,11 @@
+Rails.application.routes.draw do
+  devise_for :users, controllers: { 'registrations': 'registrations' }
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
+  end
+  resources :posts
```

Comment:
The routes.rb for post, needs to write like:  `resources :posts, only: [:new, :create, :index]`

Reply:
I have added delete and edit functionality as well, if I make this change, it will break whole application. So , I am adding only: [:new, :create, :index, :edit, :update] instead.

Reply:
I have simply removed additional features from UI